amplify/backend/storage/dogtable/parameters.json

```diff
@@ -0,0 +1,5 @@
+{
+    "tableName": "dogtable",
```

Comment:
@ZhuochaoZhong can you explain this to me?

Reply:
generated by amplify when you create a storage resource. specified the table name and the primary key of the table.